apps/kg/dataloader/sampler.py

```diff
@@ -172,7 +172,7 @@ def __init__(self, g, edges, batch_size, neg_sample_size, mode, num_workers):
                                    num_workers=num_workers,
                                    shuffle=False,
                                    exclude_positive=False,
-                                   relations=g.edata['id'],
+                                   relations=g.edges[:].data['id'],
```

Comment:
what is the difference?

Reply:
Graph store does not support edata and ndata (all edges/nodes) for now, so I just hacked the code like that.